client/src/components/shared/timers/TimerAaron.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```

Comment:
Same thing here. Rename file to something more generic.

Reply:
Also, you could probably delete the the other timer/countdown files we aren't using, if you haven't already.